docs/extend/modular-resources/create/_index.md

```diff
@@ -80,7 +81,7 @@ This main program is the "entry point" to your module.
 
 ### Code a new resource model
 
-The following example module registers a modular resource implementing Viam's built-in [Base API](/components/base/#api) [(rdk:service:base)](/extend/modular-resources/key-concepts/#models) as a new model, `"mybase"`:
+The following example module registers a modular resource implementing Viam's built-in [Base API](/components/base/#api) [(rdk:service:base)](/extend/modular-resources/key-concepts/#models) as a new model, `"mybase"`, using the model family `acme:demo:mybase`. For more information on model naming see [Model namespace](/extend/modular-resources/key-concepts/#namespace-1).
```

Comment:
```suggestion
The following example module registers a modular resource implementing Viam's built-in [Base API](/components/base/#api) [(rdk:service:base)](/extend/modular-resources/key-concepts/#models) as a new model, `"mybase"`, using the model family `acme:demo:mybase`.
For more information on model naming see [Model Namespace](/extend/modular-resources/key-concepts/#namespace-1).
```
another newline and also opt capitalize "namespace" because I think it looks less weird in this link's context